content/blog/pulumi-components/index.md

```diff
@@ -605,6 +605,98 @@ runtime: yaml
 
 For other languages, you'd use the appropriate runtime value. This file is the key that enables Pulumi to generate SDKs in other languages from your component code.
 
+### Define an Entry Point
+
+The entrypoint analyzes components to automatically build a schema, and interact with the Pulumi engine to mange the component lifecycle.
+
+{{< chooser language "typescript,python,go,csharp,java" >}}
+
+{{% choosable language typescript %}}
+
+Not required for TypeScript.
+
+{{% /choosable %}}
+{{% choosable language python %}}
+
+1. Create a `_main_.py` file in your component directory
+2. In the `main` function, add a call to `component_provider_host`, specifying a list of components for the `components` argument
+
+```python
+from pulumi.provider.experimental import Metadata, component_provider_host
+from staticpage import MyComponent
+
+if __name__ == "__main__":
+    component_provider_host(name="python-components", components=[MyComponent])
+```
+
+{{% /choosable %}}
+{{% choosable language go %}}
+
+1. Define a `main.go` file
+2. Declare an instance of `NewProviderBuilder`,  passing in a name, namespace and the components being built
+
+```go
+package main
+
+import (
+    "github.com/pulumi/pulumi-go-provider/infer"
+)
+
+func main() {
+    err := infer.NewProviderBuilder().
+            WithName("go-components").
+            WithNamespace("your-org-name").
```

Comment:
Minor but could drop `namespace` from this example.

Reply:
I grabbed this from docs so leaving now but we could drop in both if we want!